lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -54,7 +54,7 @@ class DiscussionTab(EnrolledTab):
     """
 
     type = 'discussion'
-    title = ugettext_noop('Discussion')
+    title = pgettext('Courseware discussion tab', 'Discussion')
```

Comment:
Another note, the context name is a bit lengthy, try to make your work consistent with the other edX and Edraak programmers. Do a [`$ git grep pgettext`](https://github.com/edx/edx-platform/search?l=Python&q=pgettext&type=Code&utf8=%E2%9C%93) to get an idea how other people write context names.